docs/serving/autoscaling/scale-bounds.md

```diff
@@ -217,6 +213,59 @@ delay period.
           scale-down-delay: "15m"
     ```
 
+## Stable window
+
+The stable window defines minimal time that passes between decisions to change the scale of a revision. [Panic mode](kpa-specific.md) can override this setting.
+
+* **Global key:** `stable-window`
+* **Per-revision annotation key:** `autoscaling.knative.dev/window`
+* **Possible values:** Duration, `6s` <= value <= `1h`
+* **Default:** `60s`
+
+!!! note
+    During scale down, the last Replica will only be removed after there has not been any traffic to the Revision for the entire duration of the stable window.
```

Comment:
I _think_ this is true, in that if there hasn't been any traffic during the stable window the average metric will not be 0, so we won't scale down, but I'm not completely sure that we guarantee this e.g. if [ExponentialWeightedBuckets](https://github.com/knative/serving/blob/main/pkg/autoscaler/aggregation/bucketing.go#L150) were used I wonder if the average might go to zero faster. I'm probably being too pedantic here, though 😅.

Reply:
@julz Do you think slightly changing the wording here would make sense - maybe to something less definite like:
> During scale down, the last Replica is usually removed after there has not been any traffic to the Revision for the entire duration of the stable window

Or do you think what is here now is OK?

Reply:
sorry for slow response @snneji I think I might've replied to this on mobile and forgot to hit submit 🤦; yeah I think that looks good